CalibCalorimetry/CaloTPG/src/CaloTPGTranscoderULUT.h

```diff
@@ -43,8 +43,8 @@ class CaloTPGTranscoderULUT : public CaloTPGTranscoder {
   typedef unsigned int LUT;
   typedef std::vector<double> RCTdecompression;
 
+  const HcalTopology* theTopology;
```

Comment:
it could be good to initialise it to `nullptr` in the c'tor.
Also, make sure the interface guarantees there is no null-ptr dereference.